docs/source/executing-operations/subscription-support.mdx

```diff
@@ -115,6 +116,16 @@ flowchart LR;
 
     - You can _skip_ modifying subgraph schemas that don't define any `Subscription` fields.
 
+1. If you're using Apollo Server, [update your Apollo Server instances to version 4 or later](/apollo-server/migration).
+    - Follow the Apollo Server 4 [guide for enabling subscriptions](/apollo-server/data/subscriptions#enabling-subscriptions). 
+    - Update each `npm` package that makes up Apollo Federation to version 2.4 or later:
+        - `@apollo/gateway`
+        - `@apollo/query-planner`
+        - `@apollo/subgraph`
+        - `@apollo/federation-internals`
+        - `@apollo/query-graphs`
+        - `@apollo/composition`
```

Comment:
Most of these aren't relevant to running an Apollo Server _subgraph_, and we shouldn't be recommending the installation of most of these packages either.

For Apollo Server as a subgraph, users should install `@apollo/server` and `@apollo/subgraph`.

For Apollo Server as a _gateway_, users should install `@apollo/server` and `@apollo/gateway`. However, this is a guide for router so there shouldn't be a mention of gateway here at all.

The remaining packages are all considered dependencies of `@apollo/gateway` and `@apollo/subgraph` and should not (in the majority of cases) be installed into a user's project.